docs/wmco-prerequisites.md

```diff
@@ -2,8 +2,16 @@
 Below are the Cluster and OS prerequisites for WMCO. Please also see the vSphere-specific section that 
 may be relevant.
 
-## Supported Cloud Providers based on OKD/OCP Version and WMCO version
-| Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
+## Supported Installation method
+* Installer-Provisioned Infrastructure (IPI) installation method is the recommended and supported installation method. 
+  The latter is consistent across all supported platforms. 
+
+* User-Provisioned Infrastructure (UPI) is **not supported** in any platform, the exception is Bare metal or 
+  Provider-agnostic platform (`platform: none`) where the [BYOH (Bring Your Own Host)](byoh-instance-pre-requisites.md#byoh-instance-pre-requisites) 
+  use case is supported.
```

Comment:
Consider telling what we support first e.g.
```
* User-Provisioned Infrastructure (UPI) is only supported on Bare Metal or in Provider-agnostic platform
  (`platform: none`) environments, where the [BYOH (Bring Your Own Host)](byoh-instance-pre-requisites.md#byoh-instance-pre-requisites)
  use case is available. UPI is not supported on other cloud platforms.
```

Reply:
Addressed